src/Simplex/Chat.hs

```diff
@@ -785,6 +790,19 @@ processChatCommand' vr = \case
       unzipMaybe3 :: Maybe (a, b, c) -> (Maybe a, Maybe b, Maybe c)
       unzipMaybe3 (Just (a, b, c)) = (Just a, Just b, Just c)
       unzipMaybe3 _ = (Nothing, Nothing, Nothing)
+  APICreateChatItem folderId (ComposedMessage file_ quotedItemId_ mc) -> withUser $ \user -> do
+    forM_ quotedItemId_ $ \_ -> throwError $ ChatError $ CECommandError "not supported"
+    nf <- withStore $ \db -> getNoteFolder db user folderId
+    -- TODO: assertLocalAllowed user MDSnd nf XMsgNew_
```

Comment:
XMsgNew doesn't make sense in this context, it's a protocol message. Api already requires msg content so only content items can be created
```suggestion
```